SECURITY.md

```diff
@@ -0,0 +1,35 @@
+# Security Policy
+
+The VA.gov team takes security issues in VA.gov seriously. We appreciate your efforts to responsibly 
+disclosure your findings, and will make every effort to acknowledge your contributions.
+
+## Supported Versions
+
+We release patches for security vulnerabilities. Which versions are eligible receiving such patches depend on the 
+[CVSS v3.0](https://www.first.org/cvss/calculator/3.0) Rating:
+
+| Version | Supported          |
+| ------- | ------------------ |
+| >4.0.x  | :white_check_mark: |
+| < 4.0   | :x:                |
+
+## Responsible disclosure security policy
+
+A responsible disclosure policy helps protect users of the project from publicly disclosed security vulnerabilities 
+without a fix by employing a process where vulnerabilities are first triaged in a private manner, and only publicly 
+disclosed after a reasonable time period that allows patching the vulnerability and provides an upgrade path for users.
+
+When contacting us directly via email, we will do our best efforts to respond in a reasonable time to resolve the issue. 
+
+We kindly ask you to refrain from malicious acts that put our users, the project, or any of the project’s team members 
```

Comment:
this isn't very actionable; do we have more concrete guidance on what malicious acts are/aren't? if not, let's strike this line